### DISCUSSION TITLE
Autosave in last folder

### DISCUSSION THREAD

**Discussion by kacago22** [xournalpp/xournalpp#5674] [Ideas]
I have been using xournal++ (the nightly build in windows10) for teaching math for 20+ hours a week (for over a year now).

It does not crash often, but when it does and my file is an unsaved file - then it's impossible to find the autosave file. 

Would it be possible to autosave the unsaved files in the last folder where I saved a xopp file? That way the user could find it in no time!

Thanks for considering.

**Comment by kacago22** [comment]
Up!

**Comment by PillowTrauma** [comment]
It's not what you're after, but in case you didn't know, the autosaves of unsaved .xopp go into a cache folder. You can find the path for your operating system here. Could make a desktop shortcut.

https://xournalpp.github.io/guide/file-locations/

**Reply by kacago22** [reply]
Thanks! I was able to find it and created a shortcut on the desktop! 

Now so much unsaved stuff to browse through :)
